### ISSUE TITLE
Creating groups via WEB UI

### ISSUE THREAD

**Issue by tropnikovvl** [netbirdio/dashboard#410]
**Is your feature request related to a problem? Please describe.**
At the moment, manual creation of groups is only possible through [API](https://docs.netbird.io/api/resources/groups#create-a-group), it would be great to add this functionality to WEB UI

**Describe the solution you'd like**
Add the possibility of creating groups in WEB UI

**Comment by heisbrot** [comment]
Hey @tropnikovvl,

Can you elaborate more on your use case? 

The dashboard does support the creation of groups. In every group selection, you can just type your group and select it. 
If it does not exist it will be created. 

![CleanShot 2024-09-03 at 15 23 54](https://github.com/user-attachments/assets/b3dac167-8d55-4c0c-b7ef-b0b1cc02d8da)

**Comment by tropnikovvl** [comment]
@heisbrot 
Oh my god, I completely missed this feature before.
I searched for it on this page and couldn't find it.

![Screenshot 2024-09-03 at 15 34 46](https://github.com/user-attachments/assets/7fb92c2a-191b-48ee-b304-dc1313dfa8b1)

Maybe I'm not alone, is it possible to make a button here to create groups?

**Comment by heisbrot** [comment]
Good point. I think it makes sense to create groups on that page. We will add a button in the next release.
